go/mysql/replication/replication_status.go

```diff
@@ -280,27 +285,53 @@ func ParseFilePosPrimaryStatus(resultMap map[string]string) (PrimaryStatus, erro
 }
 
 // ParseReplicationStatus parses the common (non-flavor-specific) fields of ReplicationStatus
-func ParseReplicationStatus(fields map[string]string) ReplicationStatus {
+func ParseReplicationStatus(fields map[string]string, replica bool) ReplicationStatus {
```

Comment:
IMO this parameter name was confusing. IMO this would be more clear: `replicaTerminology`.

Reply:
Fixed!